src/org/openvv/OVVAsset.as

```diff
@@ -438,6 +429,46 @@ package org.openvv {
                 _intervalTimer.addEventListener(TimerEvent.TIMER, onIntervalCheck);
                 _intervalTimer.start();
             }
+    }
+
+        private function setStage(evt:Event = null):void
+        {
+            var ad:DisplayObject = _ad as DisplayObject;
+            if(!ad) return;
+
+            ad.removeEventListener(Event.ADDED_TO_STAGE, setStage);
+            try{
+                _stage = ad.stage;
+            }
+            catch(ignore:Error){
+                //stage is inaccessible
+            }
+            if(!_stage)
+                ad.addEventListener(Event.ADDED_TO_STAGE, setStage);
+        }
+
+        private function getDisplayState(results:OVVCheck):String
+        {
+            var hasStageAccess:Boolean;
```

Comment:
In Flash, Boolean is initialized as false implicitly by default. Initializing as false is redundant.

On Jan 30, 2015, at 4:58 PM, bdementatbrightroll <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/org/openvv/OVVAsset.as<https://github.com/openvv/openvv/pull/75#discussion_r23875722>:

> +            if(!ad) return;
> +
> +            ad.removeEventListener(Event.ADDED_TO_STAGE, setStage);
> +            try{
> +                _stage = ad.stage;
> +            }
> +            catch(ignore:Error){
> +                //stage is inaccessible
> +            }
> +            if(!_stage)
> +                ad.addEventListener(Event.ADDED_TO_STAGE, setStage);
> +        }
> +
> +        private function getDisplayState(results:OVVCheck):String
> +        {
> +            var hasStageAccess:Boolean;


Could be clearer if you initialized this to false

—
Reply to this email directly or view it on GitHub<https://github.com/openvv/openvv/pull/75/files#r23875722>.